modules/ROOT/pages/policies-compare-mule3-and-mule4.adoc

```diff
@@ -0,0 +1,30 @@
+= Policies and Mule Versions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+In Mule versions 3 and 4, the policies remain the same, with a few exceptions. The following section explains the similarities and differences in these Mule versions.
+
+Policies in Mule 4
```

Comment:
There are some things missing in the table that are key differences as well:

- Mule 4 supports versioning of policies
- Mule 4 custom policies are uploaded to Exchange
- Mule 4 introduces better isolation between policies and between policies and the application
- Mule 4 introduces better error handling. 

@balbifm @mtqp  can you think of more that are not listed below?

Reply:
Just adding another bullet you mention in a following comment:
- All policies are non-blocking in Mule 4.
- Rate Limit, Rate Limit SLA and Spike Control can be ordered anywhere in the chain.

Reply:
Done! Thanks!